python/tests/unit/arcticdb/version_store/test_empty_column_type.py

```diff
@@ -15,17 +15,18 @@
 class DtypeGenerator:
     """
     Class which can generate all dtypes which ArcticDB supports. It can generate them by type category e.g. int, float,
-    etc. It can also generate a list of all available dtypes
+    etc. It can also generate a list of all available dtypes. Not all supported scalar dtypes are generated as this
+    leads to a combinatorc explosion in test case count.
```

Comment:
This description is misleading now we don't generate all types, I would say it just generates a representative subset of our supported types sufficient for testing empty column types